strapi/strapi
# Fix #14712: Query Parameter for Locale (API Documentation Plugin)

### What does it do?

Adds a query parameter for locale. 

### Why is it needed?

There is no query parameter for locale generated into the API documentation. 

### How to test it?

- Open Swagger API Documentation
- Select any GET request
- There there should be a new query parameter called `locale`

### Related issue(s)/PR(s)

#14712
